<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Task</class>
 <widget class="QWidget" name="Task">
  <property name="geometry">
   <rect>
    <x>0</x><y>0</y><width>240</width><height>120</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="lineEditTask"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonSend">
     <property name="text">
      <string>Enviar</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelTask">
     <property name="text"><string>Label Task</string></property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
